packages/svelte/tests/signals/test.ts

```diff
@@ -112,6 +112,45 @@ describe('signals', () => {
 		};
 	});
 
+	test('unowned deriveds are not added as reactions but trigger effects', () => {
```

Comment:
Is the name of the test correct? Isn't the change that we actually add the derived as a reaction? I think it makes sense: the problem is probably that we check if the derived is `UNOWNED` before executing it so if the execution changes that (connecting it) when we reach the point of the code you changes `skip_reaction` should be false (because now the derived it's not unowned anymore) but it's still true and you can check that by looking if it has reactions (or i suppose we could also check if `(reaction.f & UNOWNED) !== 0` in line?)